db/migrate/20180924140846_create_featured_profiles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateFeaturedProfiles < ActiveRecord::Migration[5.2]
+  def change
+    create_table :featured_profiles do |t|
+      t.string :title
+      t.text :description
+      t.string :profile_ids, array: true, default: []
```

Comment:
Why are the id's strings and not integers?

Reply:
@tyranja do you know how to change the datatype of strings to integer? There is a string_to_array method but no integer_to_array.

Reply:
I have found this: https://stackoverflow.com/questions/22394210/rails-migration-to-convert-string-to-integer-using-conversion

Reply:
I tried that, but get the following error `PG::CannotCoerce: ERROR:  cannot cast type character varying[] to integer`

Reply:
So, in the end I rename the old column (maybe delete it later) and add a new one with an integer array. That is accepted by PG.